.eslintrc.js

```diff
@@ -57,8 +58,15 @@ module.exports = {
     "node/no-unpublished-require": "off",
     "node/no-unsupported-features/es-syntax": "off",
 
+    "notice/notice":[ "error", { mustMatch: "This Source Code Form is subject to the terms of the Mozilla Public", "templateFile": "copyright.txt" } ],
+
+    "eol-last": "warn",
+    "eqeqeq": "off", // TODO: "warn",
     "no-unused-vars": [ "error", { vars: "all", args: "none", ignoreRestSiblings: false } ],
     "no-var": "off", // TODO: "warn",
+    "no-warning-comments": "off", // TODO: "warn",
```

Comment:
Why do we need this?

Reply:
Ah, I think this was more of a curiosity to see how many "TODO" comments we had throughout the code (and whether we should file issues for TODOs so we can track them better).
I figured ideally we'd eventually enable these rules, but they'd cause a semi-noisy diff currently so wasn't sure if we should bother or if we don't care [yet].